internal/data/block_syncer.go

```diff
@@ -172,13 +175,25 @@ func (bp BlockSyncer) parseCotaEntries(blockNumber uint64, entries []biz.Entry)
 					return kvPair, err
 				}
 				kvPair.ExtensionPairs = append(kvPair.ExtensionPairs, extensionPairs...)
+
+				subKeyPairs, err := bp.subKeyPairUsecase.ParseExtensionPair(blockNumber, entry)
```

Comment:
Not every extension is subkey and sub_type should be checked

Reply:
Ok, Updated